src/vs/workbench/contrib/testing/common/configuration.ts

```diff
@@ -37,6 +38,14 @@ export const enum DefaultGutterClickAction {
 	ContextMenu = 'contextMenu',
 }
 
+export const enum TestingCountBadge {
+	All = 'all',
```

Comment:
I'm not a big fan of having "all" as an option. The purpose of the badges is to show things that are happening (busy state when committing on SCM), or things that need to be done (unsaved files in the explorer, or extensions that need updating). Tests that exist are neither of these things. The closest analog would be if the explorer showed a badge displaying the number of files in your workspace, which would be rather odd.

Reply:
Yes, it makes sense, I totally agree with you